resources/config/varnish/fos_purge_xkey.vcl

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import xkey;
+
+sub fos_purge_recv {
+    if (req.method == "PURGE") {
+        if (!client.ip ~ invalidators) {
+            return (synth(405, "Not allowed"));
+        }
+
+        # If http header "key" is set, we assume purge is on key and you have Varnish xkey installed
```

Comment:
i assume compile would fail on `import xkey;` anyways, if xkey is not available. if that is the case, i think we can remove this comment above